targets/TARGET_Realtek/TARGET_AMEBA/TARGET_RTL8195A/device/TOOLCHAIN_ARM_STD/rtl8195a.sct

```diff
@@ -39,6 +39,14 @@ LR_RAM 0x10006000 0x6FFFF  {
   }
 }
 
+TCM 0x1FFF0000 0x10000{
+    TCM_OVERLAY 0x1FFF0000 0x10000{
```

Comment:
This is all zero init RAM right? Why does it need the load region TCM? If you move the execution region TCM_OVERLAY inside LR_RAM and remove the load region TCM then this builds.